exporter/splunkhecexporter/README.md

```diff
@@ -115,6 +119,16 @@ exporters:
     splunk_app_name: "OpenTelemetry-Collector Splunk Exporter"
     # Application version is used to track telemetry information for Splunk App's using HEC by App version.
     splunk_app_version: "v0.0.1"
+    heartbeat:
+      interval: 30s
+    telemetry:
+      enabled: true
+      override_metrics_names:
+        otelcol_exporter_splunkhec_heartbeat_sent: app_heartbeats_success_total
```

Comment:
Do you mean `otelcol_exporter_splunkhec_heartbeats_sent` instead of `otelcol_exporter_splunkhec_heartbeat_sent`? Having plural makes sense since it is a counter.

Reply:
Yes